ui/components/multichain/asset-picker-amount/asset-picker/index.scss

```diff
@@ -0,0 +1,19 @@
+.asset-picker {
+  display: flex;
+  align-items: center;
+  padding: 7px;
+  min-width: 80px;
+  height: 30px;
+  border-radius: 15px;
+  background-color: var(--color-background-alternative);
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &__symbol {
+    font-size: 11px;
+    margin-right: auto;
+    margin-left: auto;
```

Comment:
Can use style utility props for these styles. We also want to stay away from static font-sizes as this could create inconsistencies. I would suggest using the `Text` component. If the font size doesn't align with the design it's a red flag on the design

Reply:
Used `TextVariant.bodyXs` to get a small font instead.  Still need to think about how long asset symbols should be handled if the font can't get smaller.